### PR TITLE
rpc: update peer format in specification in NetInfo operation

### PR DESCRIPTION
**Pull request by tychoish**
Supports changes from https://github.com/tendermint/tendermint/pull/6820

### CODE REVIEW

File: spec/rpc/README.md [hunk 1]
```diff
@@ -187,71 +187,12 @@ curl -X POST https://localhost:26657 -d "{\"jsonrpc\":\"2.0\",\"id\":1,\"method\
...
-          "listen_addr": "tcp://0.0.0.0:26656",
-          "network": "cosmoshub-2",
-          "version": "0.32.1",
-          "channels": "4020212223303800",
-          "moniker": "moniker-node",
-          "other": {
-            "tx_index": "on",
-            "rpc_address": "tcp://0.0.0.0:26657"
-          }
-        },
```
Comment (marbar3778) [inline spec/rpc/README.md:208]: Some this information is useful for users, what is the reason for removing?

**Review by marbar3778 — CHANGES_REQUESTED** [review]
Please add some of the information like network, version, rpc back. They are useful for building tooling.

**Comment by tessr** [comment]
@marbar3778 this is just a spec change reflecting implementation changes that have already been made. Let's discuss these things either on that PR or on the originating issue, where other folks (including validators) have already chimed in.

**Comment by marbar3778** [comment]
> @marbar3778 this is just a spec change reflecting implementation changes that have already been made. Let's discuss these things either on that PR or on the originating issue, where other folks (including validators) have already chimed in.

The process has always been merge spec then tendermint repo. I didnt realize the endpoint was changing so drastically. It requires a better changelog and upgrading entry if this is how it is breaking.

**Review by marbar3778 — APPROVED** [review]
👍 

cc @alexanderbez this breaks tmcrawl and atlas.

**Pull request merged**